templates/admin/words.js.ts

```diff
@@ -4,13 +4,16 @@ import { getTranslatedSettings } from "../../src/lib/translation";
 
 export = (async answers => {
 
-	const useReact = answers.adminReact === "yes";
+	const hasTab = answers.adminFeatures && answers.adminFeatures.indexOf("tab") > -1;
+	const useAdminReact = answers.adminReact === "yes";
+	const useTabReact = hasTab && answers.tabReact === "yes";
+	const useReact = useAdminReact || useTabReact;
 
 	// Only do the adapter-specific things when not using React
 	const isAdapter = !useReact && answers.features.indexOf("adapter") > -1;
 	const isWidget = answers.features.indexOf("vis") > -1;
 
-	if (useReact && !isWidget) return;
+	if (useAdminReact && (!hasTab || useTabReact) && !isWidget) return;
```

Comment:
Not sure about this change. Doesn't line 9 and 10 make it so that thee old condition was okay?

Reply:
The problem is: useTabReact wil be false even if tab was not selected. 
We need to support the following cases:
- only Admin; use React for Admin (--> useAdminReact=true, hasTab=false, useTabReact=false) leads to no words.js
- Admin React and Tab React (--> useAdminReact=true, hasTab=true, useTabReact=true) leads to no words.js
- Admin React and Tab HTML (--> useAdminReact=true, hasTab=true, useTabReact=false) leads to a words.js
- Admin HTML and Tab React (--> useAdminReact=false, hasTab=true, useTabReact=true) leasds to a words.js
- only Admin; use HTML for Admin (--> useAdminReact=false, hasTab=false, useTabReact=false) leads to a words.js